PocketKit/Sources/PocketKit/FeatureFlagService.swift

```diff
@@ -56,12 +58,20 @@ class FeatureFlagService: NSObject, FeatureFlagServiceProtocol {
 
     /// Determine if a user is assigned to a test and a variant.
     func isAssigned(flag: CurrentFeatureFlags, variant: String?) -> Bool {
-        guard let cachedFlag = featureFlags[flag] else {
-            return false
+        func isAssignedViaUnleash(flag: CurrentFeatureFlags, variant: String?) -> Bool {
```

Comment:
Woah, we can do functions in functions in swift?

Reply:
Yep!